test/specs/java-function/function_spec.rb

```diff
@@ -1,23 +0,0 @@
-require_relative "spec_helper"
-require "digest"
-
-describe "Heroku's Java CNB" do
-  it "generates a callable salesforce function" do
-    Cutlass::App.new("simple-function").transaction do |app|
```

Comment:
I believe the fixture `test-fixtures/simple-function/` is unused after this change

Reply:
Correct, thanks! :) Will remove!

Reply:
I will keep it for now and add a smoke test to the `java-function` buildpack that will use it. Thanks for pointing this out! 💜

Reply:
Test added in [d983283](https://github.com/heroku/buildpacks-jvm/pull/472/commits/d983283192ae2795e3c8cce3bfecb9a5011cda30)